js/index.js

```diff
@@ -137,3 +138,115 @@ function creationOwlCarousel () {
             e.preventDefault();
     });
 };
+
+/**Coding section for Cards */
+if (document.readyState == 'loading') {
+    document.addEventListener('DOMContentLoaded', ready)
+} else {
+    ready();
+}
+
+function ready() {
+    let removerCartItemButtons = document.getElementsByClassName('btn-danger');
```

Comment:
are you planning to change this variable later? I don't think so)
If variable not gonna be changed => it should be a constant.